src/Payments.Mvc/Views/Shared/_Layout.cshtml

```diff
@@ -13,6 +13,7 @@
         <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-datepicker/1.8.0/css/bootstrap-datepicker.css" />
         <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-select/1.13.1/css/bootstrap-select.css">
         <link rel="stylesheet" href="https://cdn.datatables.net/1.10.16/css/dataTables.bootstrap4.css" />
+        <link rel="stylesheet" type="text/css" href="../../ClientApp/css/queries.css">

```

Comment:
@NabilFurmoli This will only work in development but more importantly, css should be integrated into SASS like all of the other styling for this app.  You'll need to change your css file to a SASS file.  If you follow the lead of the other files like the _navigation.sass file, it'll all get bundled and you can dump this line